pkg/network/domainspec/generators.go

```diff
@@ -319,9 +317,6 @@ func (b *PasstLibvirtSpecGenerator) Generate() error {
 	ports := b.generatePorts()
 	args := append([]string{"--runas", "107", "-e"}, ports...)
 	cmd := exec.Command("/usr/bin/passt", args...)
-	cmd.SysProcAttr = &syscall.SysProcAttr{
-		AmbientCaps: []uintptr{unix.CAP_NET_BIND_SERVICE},
-	}
```

Comment:
maybe we should leave this for now.

Reply:
Ah, right, because on d/s, we're not using Bazel

Reply:
I am a little bit confused. Why do we need it now?

Reply:
David pointed out, that we're building d/s Openshift Virtualization in a different way, we're not using bazel, so specifying this in the Bazel file **only** wouln't work on d/s, isn't that correct?

Reply:
I am not following. This PR is not touching build configuration so this solution is vendor agnostic. Enabling the `sysctl` means you don't need this anymore.

Reply:
i don't think the code highlighted here is exactly what I commented on. There was some sort of build change occurring in a previous  revision where I believe something else was altered. I can't remember now what it was.

> Enabling the sysctl means you don't need this anymore.

the sysctl for unpriv port range is only being applied when passt is in use. anything needing dhcp still needs the cap, unless we use the sysctl call for all network bindings.

Reply:
At first I thought that it's preferred to have the CAP_NET_BIND_SERVICE capability, instead of configuring the unprivileged_port_start to 0.
Which is why I kept the original capability and added the sysctl.

Reply:
> i don't think the code highlighted here is exactly what I commented on. There was some sort of build change occurring in a previous revision where I believe something else was altered. I can't remember now what it was.

That must be what confused me.

> 
> > Enabling the sysctl means you don't need this anymore.
> 
> the sysctl for unpriv port range is only being applied when passt is in use. anything needing dhcp still needs the cap, unless we use the sysctl call for all network bindings.

Absolutely and I would be for the sysctl for all bindings.